hypervisor.go

```diff
@@ -211,5 +211,7 @@ type hypervisor interface {
 	pausePod() error
 	resumePod() error
 	addDevice(devInfo interface{}, devType deviceType) error
+	hotplugDevice(devInfo interface{}, devType deviceType) error
```

Comment:
The obvious question is: "where is the corresponding `hotplugRemoveDevice`?". ciao supports it, so shouldn't we? If so, this should be named something like `hotplugAddDevice` instead.

Reply:
Addresses this one as well. hotplugRemoveDevice is now called while stopping a container.